### DISCUSSION TITLE
Request: Improve font usage

### DISCUSSION THREAD

**Discussion by ilspiert** [Azure/api-management-developer-portal#2552] [Feature request]
To better customize the Developer Portal to my cooperate CI I need more options to configure fonts:
1) Be able to select from the font list kind of standard fonts like Helvetica, Arial, sans-serif, etc. I would expect to have a textbox where this could be entered.
2) Able to define font-familys. Should be a combination of fonts I've uploaded and a list of standard fonts mentioned in 1)

**Comment by azaslonov** [comment]
Hi @fsettele, #2 is coming in Oct (we have similar request #924). 

Regarding #1, I agree on standard fonts. Can you please elaborate on "textbox where this could be entered", not sure I understand what you mean.

**Comment by ilspiert** [comment]
Hi @azaslonov, thanks for your response. According to 1) I meant, that there a two possibilities to make standard-font and font-familly work.
1) make standard-fonts available in the font list, but then is there always a font which is missing.
2) having an option like "add standard font" which opens a little window with a textbox where the user can name and add standard fonts to the list - to keep the list clean this items should be deleteable as well and kinf od marked, that these are not-default fonts - btw. font names should support unicode text

And to make font-family work it would be cool to have on the "variantion" > Typography just to have the possibility to add multible fonts.

**Comment by azaslonov** [comment]
Partially fixed: font families are now supported.

**Comment by rsch136** [comment]
Really glad that font families are now supported and that it is possible to upload own fonts. However, how do I access the different font weights in the typography pane?
